tests/integrations.rs

```diff
@@ -65,6 +65,11 @@ fn _0_ci_setup() {
     if env::var("CI").is_ok() && env::var("LOG_FILE").is_ok() {
         self::util::init_log();
     }
+    if env::var("PANIC_ABORT").is_ok() {
+        // Panics as aborts, it's helpful for debugging,
```

Comment:
there are two same comments, I prefer moving these to the function for the explanation.